packages/lending/src/protocols/compound-v3/lending-protocol.ts

```diff
@@ -313,11 +320,17 @@ export class LendingProtocol extends Protocol {
       },
     ];
     for (const { token, borrowCollateralFactor, liquidateCollateralFactor, supplyCap, totalSupply } of assets) {
+      const apy = '0';
+      const lstApy = lstTokenAPYMap[token.address.toLowerCase()] || '0';
+      const grossApy = lstApy === '0' ? apy : BigNumberJS(apy).plus(lstApy).toString();
```

Comment:
Since apy = 0 in Ln.323, do we still need to do the `plus` operation here?

Reply:
fixed